Mon 4834 educate user menus access (#8317)
centreon/centreon: doc/en/administration_guide/acl.rst

* Level 4  context menus (Monitoring > Services > By Hosts / Details > Problems)

.. note::
    By default, access is **Read Only**. If you want to allow your users to modify the configuration, you will need to
    select the **Read / Write** option for each submenu.

.. note::

>     To access to a level of menu ‘n-1’, the user must have access to the menu of level ‘n’ otherwise he will not be able

Reviewer: ```suggestion
    To access an ‘n-1’ menu level, the user must have access to the ‘n’ menu level, otherwise he will not be able
```

>     to view the menu via the interface. If this is not the case the user will have to access directly to the page

Reviewer: ```suggestion
    to view the menu via the interface. If this is not the case, the user will have to directly access the page
```

>     concerned via a direct link (autologin, etc.).

Reviewer: ```suggestion
     via a direct link (autologin, etc.).
```

.. warning::

>     Accessing the command editing menu, as well as accessing the SNMP trap editing menu can be very dangerous. Indeed,

Reviewer 2: ```suggestion
    Accessing the command editing menu, as well as accessing the SNMP trap editing menu can be harmfull. Indeed,
```

>     the privileged user can create commands that can allow the creation of security breaches (RCE). Only give this

Reviewer: ```suggestion
    the privileged user can create commands, which may lead to the creation of security breaches (RCE). Only give this
```
